src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputFormattedOrderingAndSpacing1.java

```diff
@@ -0,0 +1,43 @@
+package com.google.checkstyle.test.chapter3filestructure.rule333orderingandspacing;
+
+import static java.awt.Button.ABORT;
+import static java.io.File.createTempFile;
+import static javax.swing.WindowConstants.*; // violation '.*' form of import should be avoided'
+
+import com.google.common.base.Ascii;
+import java.awt.*; // violation 'Using the '.*' form of import should be avoided'
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Reader;
+import javax.swing.*; // violation 'Using the '.*' form of import should be avoided'
+
+/** Some javadoc. */
+public class InputFormattedOrderingAndSpacing1 {
+  /** some javadoc. */
+  public static void main(String[] args) {
```

Comment:
Please do us a favor to update original files
Like https://github.com/checkstyle/checkstyle/blob/master/src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule333orderingandspacing/InputOrderingAndSpacing1.java

To have code in class.
To let us and other in future see that difference between files a minimum

Reply:
I have formatted all the original files as well: 

https://github.com/checkstyle/checkstyle/pull/15557/files#diff-0df76b63da4b6c930800c10cee1884f054dd7f8e7d1d2ea0e6da3610319ee896

https://github.com/checkstyle/checkstyle/pull/15557/files#diff-a28667b832aebc07211c388bc72140492fc0fa8ab6f7e93b57e4cb4d75c82f1c

https://github.com/checkstyle/checkstyle/pull/15557/files#diff-a5337bee4a714ff79e55314968a2c19e2c9f04380defad6024a84a9a50f49759

and so on

Reply:
please make no diff in bodies of classes:
![image](https://github.com/user-attachments/assets/4f837dbc-7945-4c2b-88a3-c98cead4c484)

Reply:
comment is different,
block of code is missed.

Reply:
thanks for pointing this out, these were some extra instances which were unnecessary, I have removed them now.